d2cli/main.go

```diff
@@ -186,13 +187,16 @@ func Run(ctx context.Context, ms *xmain.State) (err error) {
 			inputPath = filepath.Join(inputPath, "index.d2")
 		}
 	}
+	outputFormat := getExportExtension(outputPath)
+	if outputFormat == PPT {
```

Comment:
ppt is not supported, it's wrong to shoehorn it into that list of supported extensions.

just do
`filepath.Ext == .ppt`